tests/integration/helpers.bash

```diff
@@ -349,7 +353,7 @@ function requires() {
 		local skip_me
 		case $var in
 		criu)
-			if [ -n "$HAVE_CRIU" ]; then
+			if [ ! -v HAVE_CRIU ]; then
```

Comment:
So, this fixes the bug of not running criu tests; the old check for `-n "$HAVE_CRIU`, introduced by commit 6e1d476aa, is wrong, it should have been `-z "$HAVE_CRIU"`. This is also fixed by #3380.